pom.xml

```diff
@@ -216,8 +217,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.2</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
```

Comment:
Looks like this is giving error on Travis: https://travis-ci.org/qubole/quark/jobs/105388946